pages/learn/manage/actions.md

```diff
@@ -127,7 +127,7 @@ Applications with all their associated devices, releases and members can be tran
 Only organization [administrators][administrator] can initiate and complete application transfers. You must coordinate with one of the receiving organization's administrators to perform the following actions:
 
 1. Take note of the application name in the **source** organization and your balenaCloud username (*in the top-right drop-down menu*).
-2. Ask an administrator of the **target** balenaCloud organization to create a new empty application using the same application name.
+2. Ask an administrator of the **target** balenaCloud organization to create a new empty application of [any type][app-types] using the same application name.
```

Comment:
Since this is non-mandatory information, I would just added it as a parenthesized suffix like:
```suggestion
2. Ask an administrator of the **target** balenaCloud organization to create a new empty application using the same application name (the [application type][app-types] doesn't need to match).
```
Other than that LGTM :+1: